service/assessment/assessment.go

```diff
@@ -107,7 +105,8 @@ type Service struct {
 	authorizer api.Authorizer
 
 	// grpcOpts contains additional gRPC options that will be appended to all gRPC dial calls. Useful for testing.
-	grpcOpts []grpc.DialOption
+	grpcOptsEvidenceStore []grpc.DialOption
```

Comment:
Is it really needed to split this?

Reply:
I've done that for a better testing coverage :) And I thought we might want different grpc DialOptions for different components.

Reply:
TODO: Combine WithAdditionalGRPCOptsXXX and WithXXXAddress.

Reply:
Another suggestion might be to introduce an (internal) type that also stores them together in the `Service` like 
```
type grpcTarget struct {
	target string
	opts   []grpc.DialOption
}
```